### DISCUSSION TITLE
ArrayTbale error 样式异常

### DISCUSSION THREAD

**Discussion by PlutoCA** [alibaba/formily#2392] [Q&A 2.x] [answered]
![image](https://user-images.githubusercontent.com/31099607/140052902-12115fa5-86e9-404a-b031-5e251513c2ba.png)
[demo](https://codesandbox.io/s/68kgv)

最后一行错误提示浮窗被挡住了

**Comment by janryWang** [comment, marked as answer]
这个没办法

**Reply by PlutoCA** [reply]
![image](https://user-images.githubusercontent.com/31099607/140250375-8f910869-928b-4c6c-89ca-318146c3f726.png)
[link](https://procomponents.ant.design/components/editable-table#%E5%AE%9E%E6%97%B6%E4%BF%9D%E5%AD%98%E7%9A%84%E7%BC%96%E8%BE%91%E8%A1%A8%E6%A0%BC)
这种浮窗的方案咋样

**Reply by PlutoCA** [reply]
https://github.com/alibaba/formily/blob/fb217f2aba6886810dc76aa05db4cdaff1b1c21c/packages/antd/src/form-item/index.tsx#L179
[docs](https://antd.formilyjs.org/zh-CN/components/form-item#%E5%8F%8D%E9%A6%88%E4%BF%A1%E6%81%AF%E5%AE%9A%E5%88%B6%E6%A1%88%E4%BE%8B)
已经支持了😂， 抱歉，应该再找找文档的
不过目前有办法默认设置整个表格都使用popover么？ 而不用一个个去设置列属性

**Reply by janryWang** [reply]
FormLayout可以批量设置

**Reply by PlutoCA** [reply]
[https://codesandbox.io/s/cool-nobel-vsn1u?file=/App.tsx](https://codesandbox.io/s/cool-nobel-vsn1u?file=/App.tsx)
我这边用layout 包住了table ，但是table 里面的表单好像木有生效

**Reply by janryWang** [reply]
FormLayout配置shallow为false，默认是浅传递的

**Reply by PlutoCA** [reply]
ok 可以了 感谢
